Perf-Benchmarks/AKS-Perf-Benchmarks.md

```diff
@@ -1,7 +1,6 @@
 
 This page documents the test benchmarks for node sclaing latency as conducted by the AKS team. Your experience may vary slightly based on the below listed factors, in case you see significant variance in your own test runs or production environment please create an issue in this project and engage with AKS product Group.
 
-# 11/06/23 test results 
 ### AKS 50 Node Scale Out
 Below are the configurations used to conduct these benchmark tests
```

Comment:
We can probably give some more details on what these mean...how the latency is defined and more configuration details if applicable